lisp/ess-r-flymake.el

```diff
@@ -163,9 +163,13 @@ into a list and call REPORT-FN on it."
        for msg = (match-string 4)
        for (beg . end) = (let ((line (string-to-number (match-string 1)))
                                (col (string-to-number (match-string 2))))
-                           (flymake-diag-region src-buffer line col))
+                           (if (fboundp 'flymake-diag-region)
```

Comment:
This change isn't necessary. This file assumes Emacs 26+

Reply:
But the compilation test fails without it.